chapters/operatorsandexpressions.tex

```diff
@@ -1536,4 +1536,4 @@ \subsection{Discrete-Time Expressions}\label{discrete-time-expressions}
 \subsection{Continuous-Time Expressions}\label{continuous-time-expressions}
```

Comment:
I think there could be situations where one wants to emphasize that an expression does not have any variability restriction, and then calling it _continuous-time expression_ makes it clear.  Wouldn't it also appear like a gap in the definitions to not have it defined?

Reply:
Yeah, I have very mixed feeling about Continuous-Time Expression. I like the inclusive progression: from constant, to parameter, to discrete, to continuous, and the parallel with the constant/parameter/etc variables. Except, it is misleading. There is no inclusion in the variable variability. Continuous-time variables do not include constants. And also non-discrete-time expressions do not follow the inclusion progression. They are defined by exclusion.

Maybe it's just the way the first two sentences are is what tripped me. It goes: "All expressions are continuous-time expressions.... The term non-discrete-time expression refers..." I had to read a couple of times to understand why we introduce the new term here. That's why I thought that maybe they need to be in a separate subsection all together.

Maybe it would be easier to grasp if we wrote it similar to how Discrete-time expression and others are written?

Continuous-time expressions are:

- Discrete-time expressions

- Non-discrete-time expressions (and explanation)

Should I open a separate issue about it?

Reply:
For me it would be OK to try to resolve it here.  How about at least changing the heading to _Continuous-Time 
 and Non-Discrete-Time Expressions_?  Possibly combined with some other rewording in the text beneath.

Reply:
Yes, I think that would be an improvement.

Reply:
Done.  OK now?

Reply:
Looks good. Resolving.